docs/topics/cache.txt

```diff
@@ -803,6 +803,30 @@ If you need to know whether ``add()`` stored a value in the cache, you can
 check the return value. It will return ``True`` if the value was stored,
 ``False`` otherwise.
 
+In cases where you may wish to use either a cached value or create a new cached
+value there is the ``get_or_set()`` method. It takes the same parameters as
+``get()`` but rather than simply return the default value, the default is set
```

Comment:
but the default is set as the new cache  value for that key, rather than simply returned.